src/main/java/org/mule/service/scheduler/internal/threads/SchedulerThreadPools.java

```diff
@@ -267,12 +258,9 @@ public void stop() throws MuleException, InterruptedException {
 
     final long startMillis = currentTimeMillis();
 
+    waitForExecutorTermination(startMillis);
```

Comment:
move this call down for conssitency with the previous version of the code (just in case)